### PR TITLE
Create modelTypesList.json

### PR DESCRIPTION
**Pull request by danielleali**
Model Types list will keep a mapping of model type to sample dataset and default config - we don't need to include label, description, icon etc here because they've already been added in console.

### CODE REVIEW

File: model_types/modelTypesList.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+  "modelTypes": [
+    {
+      "modelType": "synthetics",
+      "defaultConfig":  "config_templates/gretel/synthetics/tabular-lstm-evaluate.yml",
```
Comment (yamini) [inline model_types/modelTypesList.json:5]: @danielleali we'll need a `description` and icon source as well, right?

File: model_types/modelTypesList.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        "fileName": "sample-synthetic-healthcare.csv",
+        "description": "Use this sample electronic health records (EHR) dataset to synthesize an entirely new set of statistically equivalent records.",
+        "records": 9999,
+        "fields": 18,
+        "trainingTime": "6 mins",
+        "bytes": 830021
+      }
+    },
+    {
+      "title": "Amplify",
```
Comment (yamini) [inline model_types/modelTypesList.json:41]: Can we add Titles for all the model types instead of just a few?

**Review by yamini — CHANGES_REQUESTED** [review]

File: model_types/modelTypesList.json (hunk 1 above)
Comment (danielleali) [reply model_types/modelTypesList.json:5]: no, actually @thesteady already hardcoded the icon, label and description for all the model types in console, so we don't need to have that again.

File: model_types/modelTypesList.json (hunk 2 above)
Comment (danielleali) [reply model_types/modelTypesList.json:41]: Per my other comment - we actually already have all the 'titles' (called labels in console) tied to the model type - I will clean this up to remove all titles from here

Comment (yamini) [reply model_types/modelTypesList.json]: sounds good, thank you

**Review by yamini — APPROVED** [review]

**Pull request merged**